src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -1,4 +1,36 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "USERS")
+@AllArgsConstructor
+@NoArgsConstructor
 public class User {
+    @Id
+    @GeneratedValue
+    private Long id;
```

Comment:
Dla spójności kodu u wszystkich fajnie jak byś dodała adnotację:
@GeneratedValue(strategy = GenerationType.IDENTITY)
oraz
@Column(name = "user_id")

Reply:
Oraz nazwy ID zmiennych - ustalmy czy w każdej encji mamy Long id, czy Long userId, groupId, carrtId itp (To myślę że nie tylko estetyka, ale może ułatwić późniejsze wysyłanie zapytań SQL - nie trzeba będzie pamiętać czy w danej encji jest samo id czy np. userId)

Popieram  deklarację nazwy kolumny  w stylu user_id, group_id itd